crates/papyrus_protobuf/src/converters/consensus.rs

```diff
@@ -107,6 +107,49 @@ impl From<Vote> for protobuf::Vote {
 
 auto_impl_into_and_try_from_vec_u8!(Vote, protobuf::Vote);
 
+
+impl TryFrom<protobuf::StreamMessage> for StreamMessage {
+    type Error = ProtobufConversionError;
+
+    fn try_from(value: protobuf::StreamMessage) -> Result<Self, Self::Error> {
+        // let Some(message) = value.message else {
+        //     return Err(ProtobufConversionError::MissingField { field_description: "message" });
+        // };
+
+        // let Some(stream_id) = value.stream_id else {
+        //     return Err(ProtobufConversionError::MissingField { field_description: "stream_id" });
+        // };
+
+        // let Some(chunk_id) = value.chunk_id else {
+        //     return Err(ProtobufConversionError::MissingField { field_description: "chunk_id" });
+        // };
+
+        // let Some(done) = value.done else {
+        //     return Err(ProtobufConversionError::MissingField { field_description: "done" });
+        // };
```

Comment:
I am not sure why in some cases we need to check for missing fields and in other cases we can just use the values directly...